src/graph.ts

```diff
@@ -454,4 +455,44 @@ export class Graph {
       (res) => res.body.graph
     );
   }
+
+  addVertexToVertexCollection(
+    collectionName: string,
+    properties: any = {},
+    returnNew: boolean = false
+  ) {
+    return this._connection.request(
+      {
+        method: "POST",
+        path: `/_api/graph/${this.name}/vertex/${collectionName}`,
+        body: {
+          ...properties,
+        },
+        qs: {
+          returnNew,
+        },
+      },
+      (res) => res.body
+    );
+  }
+
+  addEdgeToEdgeCollection(
```

Comment:
@dlozina-macrometa Here also I guess it already exists via graph.edgeCollection(collectionName).save(...);

Reply:
⬆️ same